README.md

```diff
@@ -1,5 +1,7 @@
 # BitBoxApp
 
+![BitBoxApp Screenshot](https://shiftcrypto.ch/assets/images/app/accountssummary-with-rocket-en.png)
```

Comment:
@suriyaa please copy the PNG into this repo under `docs/assets/app-screen.png`  and link to it.

Reply:
✅ Done.